### DISCUSSION TITLE
context 及 token

### DISCUSSION THREAD

**Discussion by alexng88** [GaiZhenbiao/ChuanhuChatGPT#1032] [Q&A]
Request token (3328) large than context 512

我想请教那个位置能修改 context 长度?  例如改成 2048  
谢谢

**Comment by GaiZhenbiao** [comment]
presets.py

**Comment by alexng88** [comment]
谢回覆  但好像没有  不过最后在 python3.11/lib  llama.py 改了

我想问 用 llama2对话正常 好快回覆  但我上传文件  要用 2-3分钟才回答  这如何处理?   我 RTX40xx 8GRAM    什么规格的 GPU 才有较正常回应? (256G?)

**Reply by GaiZhenbiao** [reply]
有没有开启local_embedding？建议你开一下。

**Reply by alexng88** [reply]
咦 我以为是预设的  我是第一次接触   请问那里检查开启没有?

**Reply by alexng88** [reply]
找到了  变成 True  可是 情况一样  真的等好久才回答文件   可能硬件不久

**Reply by GaiZhenbiao** [reply]
那你可以试着
```
pip uninstall faiss-cpu
pip install faiss-gpu
```

**Reply by alexng88** [reply]
pip install pip install faiss-gpu   这句不行  我在试conda install -c pytorch.   但好像 python3.11 不能安装   晕

**Reply by GaiZhenbiao** [reply]
对不起我之前写错了

**Reply by alexng88** [reply]
没事  我可能要用 python 3.9   3.9 应该能够跑川虎GPT 吧

**Reply by GaiZhenbiao** [reply]
可以的
